app/src/main/java/ru/suslovalex/androidacademyapp/data/Cast.kt

```diff
@@ -0,0 +1,27 @@
+package ru.suslovalex.androidacademyapp.data
+
+
+import com.google.gson.annotations.SerializedName
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Cast(
```

Comment:
@AleksandrAlex I think you d not need all of these fields, please keep only the data you need